ui/raidboss/data/07-dt/raid/r6s.ts

```diff
@@ -204,6 +260,123 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'R6S Heating Up Early Warning',
+      type: 'GainsEffect',
+      netRegex: { effectId: '1166', capture: true },
+      condition: (data, matches) => data.me === matches.target && data.role !== 'healer',
+      infoText: (_data, _matches, output) => output.defamationLater!(),
+      outputStrings: {
+        defamationLater: {
+          en: 'Defamation on YOU (for later)',
+        },
+      },
+    },
+    {
+      id: 'R6S Heating Up',
+      type: 'GainsEffect',
+      netRegex: { effectId: '1166', capture: true },
+      condition: Conditions.targetIsYou(),
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 5,
+      countdownSeconds: 5,
+      alertText: (_data, _matches, output) => output.defamation!(),
+      outputStrings: {
+        defamation: Outputs.defamationOnYou,
+      },
+    },
+    {
+      // tether source is from the player to the boss
+      id: 'R6S Pudding Graf',
+      type: 'Tether',
+      netRegex: { id: ['013F', '0140'], capture: true },
+      condition: (data, matches) => data.me === matches.source,
+      infoText: (_data, matches, output) => {
+        if (matches.id === '013F')
+          return output.wingedBomb!();
+        return output.bomb!();
+      },
+      outputStrings: {
+        bomb: {
+          en: 'Drop bomb in quicksand',
+        },
+        wingedBomb: {
+          en: 'Aim bomb towards quicksand',
+        },
+      },
+    },
+    {
+      id: 'R6S Jabberwock Bind Marker',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.bindMarker, capture: true },
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Jabberwock on YOU',
+        },
+      },
+    },
+    {
+      id: 'R6S Ready Ore Not',
+      type: 'StartsUsing',
+      netRegex: { id: 'A6AA', source: 'Sugar Riot', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R6S Single Style',
+      type: 'StartsUsing',
+      netRegex: { id: '9A3D', source: 'Sugar Riot', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Look for arrows',
```

Comment:
```suggestion
          en: 'Avoid arrow lines',
```
No strong feelings here, but we might as well tell the player something definitive to do. Feel free to ignore.

Reply:
I think the `Avoid arrow lines` output text is better here.